WatchApp Extension/Controllers/CarbAndBolusFlowController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CarbAndBolusFlowController.swift
+//  WatchApp Extension
+//
+//  Created by Michael Pangburn on 4/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import WatchKit
+import SwiftUI
+import HealthKit
+import LoopCore
+import LoopKit
+
+
+final class CarbAndBolusFlowController: WKHostingController<CarbAndBolusFlow>, IdentifiableClass {
+    private var viewModel: CarbAndBolusFlowViewModel {
```

Comment:
Ah, I think I just wrote it as a computed variable to be able to access the instance member `configuration`. Switched to use `lazy` instead.

(I think the top-level `body` is computed only once since it isn't dependent upon state, but I agree it's cleaner to enforce that it's the same instance.)